Libraries/UtilLib/src/test/java/com/artardev/util/numeric/NumberUtilIsPrimeTrueTest.java

```diff
@@ -0,0 +1,29 @@
+package com.artardev.util.numeric;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import java.util.Collection;
+import java.util.List;
+
+@Ignore("T ested before and passed")
+@RunWith(Parameterized.class)
+public class NumberUtilIsPrimeTrueTest {
+    public long input;
+    @Parameterized.Parameters
+    public static Collection<Long> createData()
+    {
+        return List.of(710_584_055_392_819_667L, 1_386_437_196_678_024_971L, 19L, 71L);
+    }
+    public NumberUtilIsPrimeTrueTest(long input) {
+        this.input = input;
+    }
+    @Test
+    public void test()
+    {
+        Assert.assertTrue(NumberUtil.isPrime(input));
+    }
+}
```

Comment:
Couldn't the False and True prime test be merged within a single unit test? That way you don't need to have two separate file with a bit of redundant code...

Reply:
Perfect. Because i assume it can be reduced to something like 

```
{
    Assert.assertTrue(...)
    Assert.assertFalse(...)
}
```